.travis.yml

```diff
@@ -1,10 +1,18 @@
+stages:
+  - compile
+  # Don't run the combine stage in pull requests, because deploy is disabled there.
+  - name: combine
+    if: env(TRAVIS_PULL_REQUEST) = false
```

Comment:
This doesn't appear to be working. This conditional DSL appears to only have limited env access (per https://docs.travis-ci.com/user/conditions-v1). But, per that same link, I think `type != pull_request` should work.

Reply:
I changed it, but where did you see this not working? I looked at the build in this PR and it looks like it was excluded properly.

Reply:
It didn't run here: https://travis-ci.org/github/urbit/urbit/builds/701432802. Looks good in the latest: https://travis-ci.org/github/urbit/urbit/builds/701439095.

Reply:
I see. I didn't understand that there's a difference between the tryjob for the commit itself and the tryjob for the merge commit associated with this PR.